pages/wiki/ip-connection.md

```diff
@@ -0,0 +1,49 @@
+---
+title: IP Connection
+layout: documentation
+---
+
+Configuring an IP connection on your watch has to be done manually until a GUI settings option is available.
+On watches with supported WLan, you can enable wifi and configure the connection using <code>connmanctl</code> like described below.
+Forwarding IP requests to a connected PC via USB is another option, explained in the second paragraph.
+
+By default, there is no <code>root</code> or <code>ceres</code> password, and no firewall rules. A password can be set using the <code>passwd</code> command.
+
+
+
+# IP over WLan
+
+*****
+
+Connect to your watch using <code>ssh root@192.168.2.15</code> or <code>adb shell</code>.
+
+<pre><code># connmanctl
```

Comment:
What’s the canonical way of integrating code blocks in these documentation pages ? Some seem to only have a `<code>`, some use `<pre><code>`, some use back ticks, some use indentation etc… maybe we should uniformize that a bit ?

Reply:
I may be wrong but afaiu <pre> is for multi line <code> like ``` ?
The backticks where made possible by the move to markdown and i try to use them where possible.
But i have not fully understood where backticks/markdown works since e.g. here i hd to use <code> since backticks where not recognized *for some reason ®*